### DISCUSSION TITLE
DDT4all installation

### DISCUSSION THREAD

**Discussion by alitarasi** [cedricp/ddt4all#922] [General]
Hi everyone
 I install the latest version of DDT4all. but after installation it did not work. what should I do?

**Comment by matzel82** [comment]
What specifically ?

**Reply by alitarasi** [reply]
it could not connect to interface. but the lower version connects easily

**Reply by KarelSvo** [reply]
wifi?

**Reply by alitarasi** [reply]
yes

**Reply by KarelSvo** [reply]
https://github.com/cedricp/ddt4all/pull/659/files

**Comment by AndyIreland** [comment]
use the lower version if it lets you adjust the parameters you want to adjust - is there anything on the new version you can do which you cannot on the earlier one?
